torchvision/transforms/functional_tensor.py

```diff
@@ -18,6 +18,88 @@ def _get_image_size(img: Tensor) -> List[int]:
     raise TypeError("Unexpected type {}".format(type(img)))
 
 
+# torch.iinfo isn't scriptable so using this helper function
+# https://github.com/pytorch/pytorch/issues/41492
+def _max_value(dtype: int) -> int:
+    a = torch.tensor(2, dtype=dtype)
+    signed = 1 if torch.tensor(0, dtype=dtype).is_signed() else 0
+    bits = 1
+    max_value = torch.tensor(-signed, dtype=torch.long)
+    while(True):
+        next_value = a.pow(bits - signed).sub(1)
+        if next_value > max_value:
+            max_value = next_value
+            bits *= 2
+        else:
+            return max_value.item()
+    return max_value.item()
```

Comment:
Nice! Although I wonder if simply having a table with the min / max values wouldn't be a simpler (and more efficient) workaround while `iinfo` isn't implemented?

Reply:
It would certainly be more efficient, just slightly more work to maintain. I'm open to either way.

Reply:
Given that you already have this implementation here, let's keep it as is for now and add a TODO saying to replace it with `torch.iinfo` when it gets torchscript support.